articles/contributing-style-guide.md

```diff
@@ -108,16 +107,16 @@ We suggest:
 
 ### Entry points
 
-When defining an entry point into a Q# program, the Q# compiler recognizes the [`@EntryPoint()` attribute](xref:Microsoft.Quantum.Core.EntryPoint) rather requiring that entry points have a particular name (for example: `main`, `Main`, or `__main__`).
-That is, from the perspective of a Q# developer, entry points are ordinary operations annotated with `@EntryPoint()`.
-Moreover, Q# entry points may be entry points for an entire application (for example, in Q# standalone executable programs), or may be an interface between a Q# program and the host program for an application (for example, when using Q# with Python or .NET), such that the name "main" could be misleading when applied to a Q# entry point.
+When you define entry points for Q# programs, the Q# compiler recognizes the [`@EntryPoint()` attribute](xref:Microsoft.Quantum.Core.EntryPoint). Entry points should have a specifc name (for example: `main`, `Main`, or `__main__`).
```

Comment:
This edit inverts the meaning of the original text; Q# explicitly does not require naming entry points as `main`, `Main`, or `__main__`.

Reply:
In languages like Python, C, C++, Java, and C#, it is typical that the entry point for an application is identified by a special name like `main`, `Main`, or `__main__`. That in turn requires that users name their entry points in a special way.

Q#, by contrast, does not impose such a requirement. Entry points are denoted by `@EntryPoint`, but users are free to name those entry points whatever they want. The style guide recommends that users do so in a way that is consistent with more general guidelines for naming Q# operations (as verbs, avoiding esoteric names, and so forth).